packages/feature-qpf/test/views/quadpatternfragments/QuadPatternFragmentsRdfView-test.js

```diff
@@ -79,9 +80,9 @@ describe('QuadPatternFragmentsRdfView', function () {
 
         describe('with a non-empty triple stream that writes metadata first', function () {
           var results = AsyncIterator.fromArray([
-            { subject: 'a', predicate: 'b', object: 'c' },
-            { subject: 'a', predicate: 'd', object: 'e' },
-            { subject: 'f', predicate: 'g', object: 'h' },
+            N3.DataFactory.quad(N3.DataFactory.namedNode('a'), N3.DataFactory.namedNode('b'), N3.DataFactory.namedNode('c'), N3.DataFactory.defaultGraph()),
```

Comment:
I see you make use of this pattern quite a bit in the test.
In may make sense to simplify these with this: https://www.npmjs.com/package/rdf-quad